code/modules/spells/aoe_turf/conjure/arcane_golem.dm

```diff
@@ -72,6 +64,8 @@
 
 	if(!istype(spell_to_copy) || !istype(spell_to_copy.holder))
 		return
+	if(is_type_in_list(spell_to_copy, forbidden_spells))
```

Comment:
Bad object design encapsulation principles, and whitelist/blacklist weird comboing is difficult to represent without becoming overcomplicated. It would be unthinkable in any existing language to use typechecking for object behaviour the way DM coders do so liberally.